index.js

```diff
@@ -5,7 +5,13 @@
 module.exports = {
   name: require('./package').name,
 
-  includedCommands: function() {
+  includedCommands() {
     return require('./lib/commands');
+  },
+
+  checkDevDependencies() {
+    const VersionChecker = require('ember-cli-version-checker');
```

Comment:
@dcombs / @step2yeung  / @choheekim  this is likely causing the ember-cli-version-checker issue.

It's not clear if it is a bug in latest version-checker or a bug here.

Reply:
https://github.com/ember-cli/ember-exam/pull/136/commits/8141a5daca495f4f779012e97d6b590f8df87cdc